src/partr.c

```diff
@@ -91,6 +91,16 @@ static inline void multiq_init(void)
     unbias_cong(heap_p, &cong_unbias);
 }
 
+// Returns number of tasks contained in the queues
+JL_DLLEXPORT uint64_t jl_task_queue_length(void)
+{
+    uint64_t total = 0;
+    for (int32_t i = 0; i < heap_p; ++i) {
+        total += heaps[i].ntasks;
```

Comment:
```suggestion
        total += jl_atomic_fetch_relaxed(&heaps[i].ntasks);
```
(and the stores need to have the same change)